src/PantherTestCaseTrait.php

```diff
@@ -163,10 +163,20 @@ protected static function createPantherClient(array $options = [], array $kernel
 
         self::startWebServer($options);
 
+        $browserArguments = null;
+
+        if (\array_key_exists('browser_arguments', $options)) {
+            if (!\is_array($options['browser_arguments'])) {
+                @trigger_error('Expected key "browser_arguments" to be an array.', \E_USER_WARNING);
```

Comment:
You should throw an `\InvalidArgumentException` instead. We don't trigger errors (except for deprecation notices) in Symfony.